src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/IStringBuilderWriteable.cs

```diff
@@ -1,36 +1,22 @@
 using System.Text;
-using JetBrains.Annotations;
+using System.Threading;
 
 namespace RegressionGames.StateRecorder
 {
     public interface IStringBuilderWriteable
```

Comment:
This now does what its name says it does and there is a new IStringBuilderConverter interface encapsulating the old behaviour